pycaret/internal/tabular.py

```diff
@@ -65,7 +65,7 @@
 import warnings
 from IPython.utils import io
 import traceback
-from unittest.mock import patch
+from unittest.mock import NonCallableMock, patch
```

Comment:
This isn't used, right?
```suggestion
from unittest.mock import patch
```

Reply:
Nope